### DISCUSSION TITLE
Any advice on the AS6510T?

### DISCUSSION THREAD

**Discussion by jflo** [mafredri/asustor_as-6xxt#8] [General]
Any experience with this model? I'm trying to boot UnRaid from my USB, and for all the other models you just change the boot order in BIOS, and then you're good.

For some unfathomable reason, this unit has no video out of any kind, and the terribad ADM software doesn't give any access to the BIOS either.  

I followed most of how you got to boot from USB, however this unit doesn't seem to use grub, sooooo.... any ideas?

**Comment by mafredri** [comment]
Hey, have you tried holding the USB-copy button while booting from an USB stick plugged into the copy port (or back, can't recall if it matters)?

Assuming the USB stick is bootable and boots into a live image that automatically sets up networking, I'm guessing you should be able to connect to it once it's booted.

If that doesn't work, there are some other tricks that could be tried, but without hooking up to the serial port to see the boot process, we can only guess at what it does.

Another option would be to hook up a keyboard to one of the back USB-ports, and whilst booting it up trying a key combination that lets the BIOS boot from alternate medium. I can't remember what the keys/combos are right now, but I can check what it is on my AS62XXT in case it's the same.
